tools/recalc.py

```diff
@@ -219,25 +219,26 @@ async def recalculate_mode_scores(mode: GameMode, ctx: Context) -> None:
 
 async def main(argv: Sequence[str] | None = None) -> int:
     argv = argv if argv is not None else sys.argv[1:]
-    if len(argv) == 0:
-        argv = ["--help"]
```

Comment:
does it still behave the same way after the change here?

Reply:
Forcing that argument on makes it impossible to run it without args (which runs on all modes all calcs)

You need to add --help now